gatsby_fp/src/styles/facultyprofile.css

```diff
@@ -19,16 +19,23 @@ body {
   text-transform: none;
 }
 .img_container{
-  float: left;
-  padding-right: 2em;
+  margin-right: 2%;
 }
-.pf_field_wrapper{
+.field_wrapper_main{
```

Comment:
Where is this class being used?

Reply:
When I first started this before the flexbox, the title of education was behaving weirdly but now this is no longer needed so I got rid of it. @dmotan

Reply:
(well not really just education, the first field title that is populated)